bin/bootstrax

```diff
@@ -95,6 +95,12 @@ actions.add_argument(
 actions.add_argument(
     '--undying', action='store_true',
     help="Except any error and ignore it")
+actions.add_argument(
+    '--pyroscope', action='store_true',
+    help="Use pyroscope profiler")
+parser.add_argument(
+    '--pyro_address', type=str, default='http://pyroscope:4040',
```

Comment:
What's involved with hosting the server backend? I assume it's something we can set up ourselves on eb2.

Reply:
Its pretty lightweight so we can run it anywhere that both bootstrax and grafana (grafana in on the above ground lngs network) can access as long as that machine can spare a bit of CPU and network bandwidth.

Reply:
CPU and network are almost never the issue, if it were RAM it might have been a different story